Makefile

```diff
@@ -229,9 +229,11 @@ build-unit-tests:
 test: unit
 build-tests: build-unit-tests
 
-.PHONY: cover
-cover: TEST_FLAGS += -cover
-cover: test
+.PHONY: test-cover
+test-cover:  ## Run tests with code coverage and code generate reports
+	go test -v -coverprofile=coverage.out ./...
```

Comment:
we need coverage.out file for test coverage pipeline in test-infra

Reply:
if this file is never needed in VCS, shall we add it into `.gitignore`?

Reply:
Good catch! Added into `.gitignore`

Reply:
Also need
```
*.out
coverage.*
```

Also the coverage output

Reply:
fixed! thanks